### DISCUSSION TITLE
posts/671

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#32] [Announcements]
# posts/671

카카오는 내부 로그 수집 시스템으로 kemi-log를 운영하고 있습니다. kemi...

https://tech.kakao.com/posts/671

**Comment by livlikwav** [comment]
Fluentd 의 내부 구현 방식과 한계점,  
그리고 그걸 어떻게 해결했는지 쉽게 알 수 있었어요.
잘 봤습니다! 😃
